vendor.mod

```diff
@@ -101,7 +101,7 @@ require (
 	golang.org/x/mod v0.11.0
 	golang.org/x/net v0.17.0
 	golang.org/x/sync v0.3.0
-	golang.org/x/sys v0.13.0
+	golang.org/x/sys v0.14.1-0.20231108175955-e4099bfacb8c
```

Comment:
It looks like v0.15.0 was tagged; also might be better to update this in a separate commit